### DISCUSSION TITLE
足場の候補が複数ある場合にもっとも進行方向に向かう歩容を選ぶ方法について

### DISCUSSION THREAD

**Discussion by Naoki-Hiraoka** [YutaKojio/hrpsys-base#65] [Q&A] [answered]
上位から司令された進行方向(例えば`vel_param`)ではなく，(前回の周期で修正後の)次の目標着地位置から計算される進行方向(`cur_fs`)を使用しているのはなぜでしょうか．

https://github.com/YutaKojio/hrpsys-base/blob/49aeca84efde2d52655e3ee6b0fb0fc1219861cb/rtc/AutoBalancer/GaitGenerator.cpp#L1538

**Comment by YutaKojio** [comment, marked as answer]
cur_fsの初期値は 上位から司令された進行方向 から決まるからほぼ同じつもりでつかってたけど，
確かに，修正を重ねると一番進行方向に近いのを選んでるとは限らなくなるな
